src/backends/perf_pt/collect.c

```diff
@@ -265,7 +265,22 @@ open_perf(pid_t target_tid) {
     attr.precise_ip = 3;
 
     // Acquire file descriptor through which to talk to Intel PT.
-    ret = syscall(SYS_perf_event_open, &attr, target_tid, -1, -1, 0);
+    //
```

Comment:
Delete the blank comment line?

Reply:
It's intentional. It's spacing the following text from the above (untouched) comment line.

Reply:
Just make it a blank line then without the `//`?

Reply:
Er, well it's part of the same comment.

It's supposed to read like:
```
// Do this thing.
//
// Explain why.
// blah blah.
```

Reply:
( I don't feel strongly, so if you really want to kill the line...)

Reply:
I've never seen anyone except Knuth do this, and that whole literate programming thing never caught on ;) I'd prefer we killed either the line or at least the `//`. Sorry for nitpicking!

Reply:
What can I say. People of our calibre (me + Knuth)...

Fix coming.